yt_dlp/extractor/radiko.py

```diff
@@ -186,10 +186,12 @@ def _real_extract(self, url):
         return {
             'id': video_id,
             'title': try_call(lambda: prog.find('title').text),
+            'artist': try_call(lambda: prog.find('pfm').text),
             'description': clean_html(try_call(lambda: prog.find('info').text)),
             'uploader': try_call(lambda: station_program.find('.//name').text),
             'uploader_id': station,
             'timestamp': vid_int,
+            'duration': (unified_timestamp(radio_end, False) - unified_timestamp(radio_begin, False)),
```

Comment:
this is fatal, `unified_timestamp` can return `None`. wrap in `try_call`

Reply:
Thank you for your feedback. I fixed it.